### PR TITLE
Minor grammatical improvements

### PR DESCRIPTION
**Pull request by der-gabe**
Just a few things I noticed while reading the docs…

### CODE REVIEW

File: xml/obs_request_and_review_system.xml [hunk 1]
```diff
@@ -170,26 +170,26 @@
     certain package. Also, new maintainers of a package will see requests in
     case the old maintainer did not handle them. </para>
    <sect3>
-    <title>Manual added reviews</title>
+    <title>Manually added reviews</title>
     <para> Reviewers can be added manually by anyone involved in a request.
      This can be used to hand over a review. In that situation the new reviewer
-     needs to be added and the own review needs to be accept. The request
+     needs to be added and the old review needs to be accepted. The request
```
Comment (hennevogel) [inline xml/obs_request_and_review_system.xml:176]: This is meant to say *own*. One user adds a review for another user, then accepts his *own* review to be out of the loop.

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Comment by hennevogel** [comment]
Thank you for the contribution @der-gabe 💐

File: xml/obs_request_and_review_system.xml (hunk 1 above)
Comment (der-gabe) [reply xml/obs_request_and_review_system.xml:176]: Ah, I see now. Then it's a just a tiny bit ungrammatical… how about this:
```suggestion
     needs to be added and the original reviewer's own review needs to be accepted. The request
```
(This suggestion makes the line overly long, so don't apply it directly, but if you agree I will make the change in a separate commit and re-wrap the lines.)

Comment (der-gabe) [reply xml/obs_request_and_review_system.xml]: Sorry for the delay, I had overlooked the :+1:

**Pull request merged**